webapp/src/components/modals/create_issue/index.js

```diff
@@ -6,21 +6,23 @@ import {bindActionCreators} from 'redux';
 
 import {getPost} from 'mattermost-redux/selectors/entities/posts';
 
-import {closeCreateModal, createIssue, fetchJiraIssueMetadata} from 'actions';
-import {isCreateModalVisible, getCreateModalForPostId, getJiraIssueMetadata} from 'selectors';
+import {closeCreateModal, createIssue, fetchJiraIssueMetadata} from '../../../actions';
+import {isCreateModalVisible, getCreateModal, getJiraIssueMetadata} from '../../../selectors';
```

Comment:
Why the `../../../`? Webpack should handle resolving.

Reply:
it was giving me a warning in the file, and I want to clear those out. :)

Reply:
A warning? What warning?

Reply:
![image](https://user-images.githubusercontent.com/1490756/57550201-b1801b80-7333-11e9-92cb-5333e38fc5b4.png)

Reply:
I think it's because it's not an external module, so we should be specific about where we're getting it

Reply:
We use this kind of resolution all over the place in the webapp. I think you just need to fix your IDE.

Reply:
ok, sounds good